### PR TITLE
Added Sigmoid Function

### PR DESCRIPTION
**Pull request by kausthub-kannan**
[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### Describe your change:

- [x] Add an algorithm?
- [ ] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

### Checklist:

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [x] All new JavaScript files are placed inside an existing directory.
- [x] All filenames should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
- [x] All new algorithms have a URL in their comments that points to Wikipedia or another similar explanation.
- [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: Maths/Sigmoid.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/**
+ * @see [Wikipedia](https://en.wikipedia.org/wiki/Sigmoid_function)
+ * Calculate the Sigmoid value of a given number.
+ * @param {number} x- The input number.
+ * @returns {number} The Sigmoid value of the input.
+ */
+
+const sigmoid = (x) => {
```
Comment (appgurueu) [inline Maths/Sigmoid.js:8]: if you omit the `{}`, you can shorten this to `const sigmoid = x => 1 / (1 + Math.exp(-x))`

Comment (kausthub-kannan) [reply Maths/Sigmoid.js]: Hello, 
I have done the required reformatting of the function.

Thank You

**Review by appgurueu — COMMENTED** [review]
Code-wise fine and a useful function to have, though also algorithmically trivial and uninteresting. Not sure whether this warrants inclusion. @raklaptudirm thoughts?

**Review by appgurueu — APPROVED** [review]
basically fine, though 5 test cases for a trivial formula feel excessive

**Comment by kausthub-kannan** [comment]
> basically fine, though 5 test cases for a trivial formula feel excessive

Should I reduce the number of test cases to 3?

**Comment by raklaptudirm** [comment]
@appgurueu I am in support of stopping additions of simple formulae as algorithms, which includes this pr. What do you think?
